src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -880,4 +880,30 @@ export default class ReimbursementRequestService {
 
     return vendorTransformer(vendor);
   }
+
+  /**
+   * Deletes the vendor
+   *
+   * @param vendorId the requested vendor to be deleted
+   * @param submitter the user deleting the vendor
+   * @returns the 'deleted' vendor
+   */
+  static async deleteVendor(vendorId: string, submitter: User) {
```

Comment:
ALSO: update the getAllVendor endpoint so that it doesn't return vendors that have been deleted. just check if each vendor has a populated dateDeleted field